Lib/mimetypes.py

```diff
@@ -477,10 +477,16 @@ def _default_mime_types():
     types_map = _types_map_default = {
         '.js'     : 'text/javascript',
         '.mjs'    : 'text/javascript',
+        '.efi'    : 'applicaion/efi',
         '.epub'   : 'application/epub+zip',
         '.gz'     : 'application/gzip',
         '.json'   : 'application/json',
         '.webmanifest': 'application/manifest+json',
+        '.nb'     : 'applicaion/mathematica',
+        '.ma'     : 'applicaion/mathematica',
+        '.mb'     : 'applicaion/mathematica',
```

Comment:
Please be careful with spelling. I suggest copy/pasting from IANA next time.
```suggestion
        '.nb'     : 'application/mathematica',
        '.ma'     : 'application/mathematica',
        '.mb'     : 'application/mathematica',
```